processor/resourcedetectionprocessor/internal/system/metadata.yaml

```
@@ -19,6 +19,10 @@ resource_attributes:
     description: The os.type
     type: string
     enabled: true
+  os.version:
+    description: The os.version
+    type: string
+    enabled: true
```

Suggests turning that change off by default and turning it on as a breaking change later.
```suggestion
    enabled: false
```